src/main/java/io/fabric8/maven/docker/service/BuildXService.java

```diff
@@ -142,10 +142,12 @@ protected void buildX(List<String> buildX, String builderName, BuildDirs buildDi
         append(cmdLine, "build", "--progress=plain", "--builder", builderName, "--platform",
             String.join(",", platforms), "--tag",
             new ImageName(imageConfig.getName()).getFullName(configuredRegistry));
-        buildConfiguration.getTags().forEach(t -> {
-            cmdLine.add("--tag");
-            cmdLine.add(new ImageName(imageConfig.getName(), t).getFullName(configuredRegistry));
-        });
+        if (!buildConfiguration.skipTag()) {
+            buildConfiguration.getTags().forEach(t -> {
+                cmdLine.add("--tag");
+                cmdLine.add(new ImageName(imageConfig.getName(), t).getFullName(configuredRegistry));
+            });
+        }
```

Comment:
Is it possible to add a test case for scenario where `buildConfiguration.skipTag()` evaluates to `true`?

Reply:
I will add this test case tomorrow.